rktlet/runtime/rkt_runtime.go

```diff
@@ -186,13 +187,15 @@ func (r *RktRuntime) ListContainers(ctx context.Context, req *runtimeApi.ListCon
 			}
 
 			container := &runtimeApi.Container{
-				Id:          resp.Status.Id,
-				Metadata:    resp.Status.Metadata,
-				Image:       resp.Status.Image,
-				ImageRef:    resp.Status.ImageRef,
-				State:       resp.Status.State,
-				Labels:      resp.Status.Labels,
-				Annotations: resp.Status.Annotations,
+				Annotations:  resp.Status.Annotations,
```

Comment:
I `:sort`'d them too. The new fields are `CreatedAt` and `PodSandboxId`, which were the only ones missing (whoops!)